plugins/inputs/conntrack/conntrack.go

```diff
@@ -90,6 +101,60 @@ func (c *Conntrack) Gather(acc telegraf.Accumulator) error {
 		}
 	}
 
+	var all bool
+	var perCPU bool
+
+	for _, collect := range c.Collect {
+		if collect == "all" {
+			all = true
+		}
+		if collect == "percpu" {
+			perCPU = true
+		}
+	}
+
+	if all || perCPU {
+		stats, err := c.ps.NetConntrack(perCPU)
+		if err != nil {
+			acc.AddError(fmt.Errorf("E! failed to retrieve conntrack statistics: %v", err))
```

Comment:
```suggestion
			acc.AddError(fmt.Errorf("failed to retrieve conntrack statistics: %w", err))
```
Same for other instances. The `E!` prefix will be automatically added by Telegraf's logging framework.